src/Microsoft.Security.Utilities.Core/WellKnownRegexPatterns.cs

```diff
@@ -183,8 +183,8 @@ public static string GenerateString(string alphabet, int count)
     public const string SuffixUrlUnreserved = $"([^{RegexEncodedUrlUnreserved}+/=]|{End})";
     public const string PrefixBase62 = $"({Start}|[^{Base62}])";
     public const string SuffixBase62 = $"([^{Base62}]|{End})";
-    public const string PrefixAllBase64 = $"({Start}|[^{Base64}-_=])";
```

Comment:
>= [](http://example.com/codeflow?start=65&length=1)

Second source of bugs, we improperly forbade the equal sign as a prefix character, missing the obvious `something=secret` case (common in connection strings in storage, cosmosdb, etc.